third_party/utils/src/test/io/bazel/rulesscala/utils/Scala3CompilerUtils.scala

```diff
@@ -0,0 +1,52 @@
+package io.bazel.rulesscala.utils
+
+import dotty.tools.dotc.Compiler
+import dotty.tools.dotc.core.Contexts._
+import dotty.tools.dotc.reporting.Diagnostic
+import dotty.tools.dotc.util.SourceFile
+
+import java.nio.file.Paths
+
+object Scala3CompilerUtils {
+  def runCompiler(
+    code: String,
+    extraClasspath: List[String] = List.empty,
+  ): List[Diagnostic] = {
+    val reporter = new TestReporter()
+
+    implicit val context: FreshContext = (new ContextBase).initialCtx.fresh.setReporter(reporter)
+
+    val fullClassPath =
```

Comment:
can you comment if this order is intentional? if not, it is more efficient to build lists right to left:

i.e.
```scala
context.settings.classpath.value :: extraClasspath ::: builtinClasspaths.filterNot(_.isEmpty)
```

Reply:
Order does not matter so updated the line with your suggestion. Thanks!